src/SourceBuild/content/eng/tools/tasks/Microsoft.DotNet.SourceBuild.Tasks.LeakDetection/CustomAttributeTypeProvider.cs

```diff
@@ -0,0 +1,85 @@
+using System;
```

Comment:
Include a license header at the top of the file.

Reply:
Added in https://github.com/dotnet/installer/pull/17605/commits/2e6190436a7326641913a147d0afe7d505eefef5